pandas_gbq/gbq.py

```diff
@@ -964,6 +967,9 @@ def read_gbq(
                 'Index column "{0}" does not exist in DataFrame.'.format(index_col)
             )
 
+    # Creating an alias for col_order, which is columns
```

Comment:
Do we need a check for whether the user provides both `col_order` AND `columns`, with a warning OR error message to say that only one is allowed?

Reply:
I talked to Tim and he mentioned that we should probably fail with ValueError if both are provided, will implement this in next commit